Calculator.c

```diff
@@ -0,0 +1,101 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<ctype.h>
+
+int main()
+{
+    char s[100];
```

Comment:
Why did we specify the max length to 100. Was it specified anywhere in the assignment?